lib/index.coffee

```diff
@@ -76,15 +76,15 @@ class DockerGC
 				@currentMtimes = layer_mtimes
 
 	removeImage: (image) ->
-		if image.repoTags? and image.repoTags.length > 0
-			# Docker will complain if we delete by id an image referenced by more
-			# than one repository
-			Promise.each image.repoTags, (tag) =>
-				console.log("GC: Removing image: #{tag} (#{image.id})")
-				@docker.getImage(tag).remove(noprune: true)
-		else
-			console.log("GC: Removing image: #{image.id}")
-			@docker.getImage(image.id).remove(noprune: true)
+		return this.tryRemoveImageBy(image, image.repoTags) 	   ||
```

Comment:
Can you remove the excess whitespace before || - I assume it lines up in your editor with your settings but it definitely doesn't in a lot of other envs

Reply:
:+1: